src/DayColumn.js

```diff
@@ -129,8 +129,9 @@ class DayColumn extends React.Component {
         max={max}
         step={step}
       >
-        {this.renderEvents()}
-
+        <div className={cn('rbc-events-container', { rtl: this.props.rtl })}>
+          {this.renderEvents()}
+        </div>
```

Comment:
We may need to handle a tweak to DnD to account for the new div? maybe not

Reply:
Good point, I'll test the DnD stuff before merging.

Reply:
I only tested some basic DnD in the examples, and it worked as far as I can tell.